main.py

```diff
@@ -0,0 +1,25 @@
+import sqlite3
+from pathlib import Path
+
+from config.config import Settings
+from etl.load import create_fund_positions_table, load_csvs_into_db, run_sql_script
+
+
+def main() -> None:
+    base_dir = Path(__file__).resolve().parent
+    settings = Settings.from_yaml(base_dir / "config" / "settings.yaml")
```

Comment:
Instead of a `setting.yaml`, I would recommend using Pydantic attributes (with defaults) that can be modified from the CLI.